Vagrantfile

```diff
@@ -90,6 +90,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# Temporary fix until https://github.com/cilium/packer-ci-build/pull/278 is
+# merged. Necessary as Cilium now needs ipset for iptables masquerading.
+sudo apt-get install -y ipset
+
```

Comment:
Is there anything blocking https://github.com/cilium/packer-ci-build/pull/278?

Reply:
No. I'd just rather have this merged before updating all VM images, or at least be fairly certain this PR will be accepted :slightly_smiling_face: